makepanda/makepanda.py

```diff
@@ -438,7 +438,7 @@ def parseopts(args):
 
     PLATFORM = 'macosx-{0}.{1}-{2}'.format(osxver[0], osxver[1], arch_tag)
 
-elif target == 'linux' and os.path.isfile("/lib/libc-2.5.so") and os.path.isdir("/opt/python"):
+elif target == 'linux' and (os.path.isfile("/lib/libc-2.5.so") or os.path.isfile("/lib64/libc-2.5.so")) and os.path.isdir("/opt/python"):
```

Comment:
This condition is very long and hard to understand and comprehend. Maybe it's better to put it into a separate method?